docs/track.rst

```diff
@@ -35,23 +35,37 @@ Custom Track Repositories
 
 Alternatively, you can store Rally tracks also in a dedicated git repository which we call a "track repository". Rally provides a default track repository that is hosted on `Github <https://github.com/elastic/rally-tracks>`_. You can also add your own track repositories although this requires a bit of additional work. First of all, track repositories need to be managed by git. The reason is that Rally can benchmark multiple versions of Elasticsearch and we use git branches in the track repository to determine the best match for each track (based on the command line parameter ``--distribution-version``). The versioning scheme is as follows:
 
-* The `master` branch needs to work with the latest `master` branch of Elasticsearch.
+* The ``master`` branch needs to work with the latest ``master`` branch of Elasticsearch.
 * All other branches need to match the version scheme of Elasticsearch, i.e. ``MAJOR.MINOR.PATCH-SUFFIX`` where all parts except ``MAJOR`` are optional.
 
-.. _track-repositories-fall-back-logic:
+.. _track-repositories-branch-logic:
 
-Rally implements a fallback logic in order of specificity up to the minor version level, so you don't need to define a branch for each patch release of Elasticsearch.
+When a track repository has several branches, Rally will pick the most appropriate branch, depending on the Elasticsearch version to be benchmarked, using a match logic in the following order:
 
-Assuming the track repository has several branches, the order is:
+#. *Exact match major.minor.patch-extension_label* (e.g. ``7.0.0-beta1``)
```

Comment:
Nit: A few lines above in `MAJOR.MINOR.PATCH-SUFFIX` we call this `SUFFIX` but here we call it `extension_label`. We should align the names.

Reply:
Agreed. Fixed in 3a4ca085b156c804ba042fac6583f2c8c2c98571